src/doc/trpl/ownership.md

```diff
@@ -121,9 +121,9 @@ let v2 = v;
 ```
 
 The first line creates some data for the vector on the [stack][sh], `v`. The
-vector’s data, however, is stored on the [heap][sh], and so it contains a
-pointer to that data. When we move `v` to `v2`, it creates a copy of that data,
-for `v2`. Which would mean two pointers to the contents of the vector on the
+vector’s data, however, is stored on the [heap][sh], and so the vector contains a
+pointer to that data. When we move `v` to `v2`, it creates a copy of that pointer,
+for `v2`. This would mean two pointers to the content of the vector on the
```

Comment:
I suggest "Which means that there would be two pointers to". "This" conflicts a bit with very similar phrase in the next sentence. The form of "means" refers to another Present Simple in the previous sentence. I will review the entire paragraph soon.